app/login/components/profile-tables.tsx

```diff
@@ -289,12 +289,12 @@ const ProfileVisualizationsRow = (props: ProfileVisualizationsRowProps) => {
         },
       }}
     >
-      <TableCell width={80}>
+      <TableCell width="10%">
         <Typography variant="body2">
           {config.data.chartConfigs.length > 1 ? "multi" : "single"}
         </Typography>
       </TableCell>
-      <TableCell width="auto" sx={{ maxWidth: 320 }}>
+      <TableCell width="35%">
```

Comment:
Not sure if important, but I think the sum of cell widths now equals 110%? :D

Reply:
🧠